src/styles/main.scss

```
@@ -0,0 +1,101 @@
+@import "./utils/variables";
+
+.body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  min-height: 100vh;
+
+  font-family: "Audiowide", cursive;
+  background-color: #000;
+}
+
+.stop-watch {
+  min-width: $sw-size;
+  min-height: $sw-size;
+
+  border-radius: 50%;
+  background: radial-gradient(#000 #{$sw-size / 3}, #e76033);
+
+  &__dial {
+    position: relative;
+  }
+
+  &__dial-number {
+    position: absolute;
+    transform: translate(-50%, -50%);
+
+    font-size: $sw-size / 8;
+    text-align: center;
+    text-shadow: 0 0 3px #ff0;
+
+    @each $name, $position in $number-alignment-x {
+      &:nth-child(#{$name}) {
+        top: $position;
+      }
+    }
+
+    @each $name, $position in $number-alignment-y {
+      &:nth-child(#{$name}) {
+        left: $position;
+      }
+    }
+  }
+
+  &__hands {
+    position: relative;
+  }
+
+  &__hand-minutes,
+  &__hand-seconds {
+    position: absolute;
+    left: $sw-size / 2 - $sw-size / 100;
+    bottom: -$sw-size / 2;
+
+    width: $sw-size / 50;
+    height: $sw-size * 0.3;
+
+    border-radius: 40% 40% 0 0;
+    background-color: #87ceeb;
+
+    animation: rotate 3600s linear infinite;
+    transform-origin: 50% 100%;
+  }
+  
+  &__hand-seconds {
+    left: $sw-size / 2 - $sw-size / 200;
+    
+    width: $sw-size / 100;
+    height: $sw-size * 0.4;
+
+    background-color: #ff0;
+    
+    animation: rotate 60s infinite steps(60);
+    
+    &::after {
+      display: block;
+      position: absolute;
+      left: 50%;
+      bottom: 0;
+      transform: translate(-50%, 50%);
+
+      width: $sw-size / 18;
+      height: $sw-size / 18;
+
+      content: "";
+      background-color: #000;
+      border-radius: 50%;
+      box-shadow: 0 0 6px 3px #ff0;
+    }
+  }
+}
+
+@keyframes rotate {
+  from {
+    transform: rotate(0);
+  }
+  to {
+    transform: rotate(360deg);

```

also, you can use [turn](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate)

```suggestion
    transform: rotate(1turn);
```